packages/@sanity/cli/src/commands/blueprints/initBlueprintsCommand.ts

```diff
@@ -1,95 +1,77 @@
-import {join} from 'node:path'
-import {cwd} from 'node:process'
-
 import {type CliCommandDefinition} from '../../types'
 
 const helpText = `
-Examples
-  # Create a new Blueprint manifest file
+Arguments
+  <dir>  Path to initialize the Blueprint in
```

Comment:
I think if dir is optional, the convention is to use square brackets
```suggestion
  [dir]  Path to initialize the Blueprint in
```